released/packages/coq-unicoq/coq-unicoq.1.6+8.15/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+maintainer: "matthieu.sozeau@inria.fr"
+authors: [ "Matthieu Sozeau <matthieu.sozeau@inria.fr>" "Beta Ziliani <beta@mpi-sws.org>" ]
+dev-repo: "git+https://github.com/unicoq/unicoq.git"
+homepage: "https://github.com/unicoq/unicoq"
+bug-reports: "https://github.com/unicoq/unicoq/issues"
+license: "MIT"
+build: [
+  ["coq_makefile" "-f" "_CoqProject" "-o" "Makefile"]
```

Comment:
Would be nice to have a proper top-level makefile in this project, this feels brittle in the long run. But let's merge anyway.

Reply:
@beta-ziliani : I tend to agree with @palmskog 's comment. It is common to have a top level makefile which abstracts from internals on how it really works.